scripts/utils/trading_strategy_helpers.js

```diff
@@ -371,21 +375,20 @@ const formatDepositString = function (depositsAsJsonString) {
 
 /**
  * Batches together a collection of transfer-related transaction information.
- * Particularly, the resulting transaction is that of transfering all sufficient funds from fleetOwner
- * to its subSafes, then approving and depositing those same tokens into BatchExchange on behalf of each subSafe. * Batches together a collection of order placements on BatchExchange
- * on behalf of a fleet of safes owned by a single "Master Safe"
- * @param {integer} fleetSize Size of the fleet
- * @param {Address} masterSafeAddress Address of the master safe owning the fleet
+ * Particularly, the resulting transaction is that of transfering all sufficient funds from master
+ * to its brackets, then approving and depositing those same tokens into BatchExchange on behalf of each bracket.
+ * @param {Address} masterAddress Address of the master safe owning the brackets
+ * @param {Address[]} bracketAddresses list of bracket addresses that need the deposit
  * @param {Address} stableTokenAddress one token to be traded in bracket strategy
- * @param {Address} targetTokenAddress second token to be traded in bracket strategy
  * @param {number} investmentStableToken Amount of stable tokens to be invested (in total)
+ * @param {Address} targetTokenAddress second token to be traded in bracket strategy
  * @param {number} investmentStableToken Amount of target tokens to be invested (in total)
  * @param {bool} storeDepositsAsFile whether to write the executed deposits to a file (defaults to false)
  * @return {Transaction} all the relevant transaction information to be used when submitting to the Gnosis Safe Multi-Sig
  */
-const buildTransferApproveDepositTransaction = async function(
-  masterSafeAddress,
-  slaves,
+const buildTransferApproveDepositFromOrders = async function(
```

Comment:
Maybe the two function `buildTransferApproveDeposit` and `buildTransferApproveDepositFromOrders` could be renamed to `buildTransferApproveDepositWithStandartFunding` and `buildTransferApproveDepositWithCustomFunding`

would you think that this is easier to understand?

Reply:
I think `buildTransferApproveDepositFromOrders` is more specific than `buildTransferApproveDepositWithCustomFunding` so I would not change it.
However i also don't like the current combination of the two names. What about renaming `buildTransferApproveDeposit` to a more specific `buildTransferApproveDepositFromList`?